cli/azd/pkg/project/service_target_dotnet_containerapp.go

```diff
@@ -455,6 +450,28 @@ func (fns *containerAppTemplateManifestFuncs) ConnectionString(name string) (str
 	case "azure.sql.v0", "sqlserver.server.v0":
 		return fns.sqlConnectionString(name, "")
 	case "azure.sql.database.v0", "sqlserver.database.v0":
+		parentResource := resource.Parent
+		if parentResource == nil || *parentResource == "" {
+			return "", fmt.Errorf("parent resource not found for db: %s", name)
+		}
+
+		parent := fns.manifest.Resources[*parentResource]
+		if parent.Type == "container.v0" {
+			rawConnectionString := strings.Split(strings.TrimRight(*parent.ConnectionString, ";"), ";")
```

Comment:
What is the most right entry here that we're trying to remove? I wonder if it's more resilient to parse the semi-colon delimited key-value pairs and remove the unwanted entry.

Reply:
The `;` at the end of the connectionString is adding an empty string to the list of parts.  

Since I am doing `return strings.Join(rawConnectionString, ";"), nil` at the end, I would be adding an extra `;` somewhere in the middle.

Would you prefer to create the list (with one empty string) and then remove the empty string from the list? (either with a new list or mutating the same one) ?

Reply:
I see. I think if we're simply appending to the end

```golang
if len(rawConnectionString) > 0 && rawConnectionString[len(rawConnectionString)-1] != ";" {
  rawConnectionString = append(rawConnectionString, ";")
}
rawConnectionString = append(rawConnectionString, "Database=xyz")
```